webapp/src/main/java/io/github/microcks/web/RestController.java

```diff
@@ -143,7 +143,7 @@ public ResponseEntity<?> execute(
                if (operation.getResourcePaths() != null) {
                   // Produce a matching regexp removing {part} and :part from pattern.
                   String operationPattern = getURIPattern(operation.getName());
-                  operationPattern = operationPattern.replaceAll("\\{.+\\}", "(.)+");
+                  operationPattern = operationPattern.replaceAll("\\{.+\\}", "([^/])+");
```

Comment:
Shouldn't we have something similar on next line too?

Reply:
yeah, thats probably true